example/ios/Flutter/Flutter.podspec

```diff
@@ -0,0 +1,18 @@
+#
+# This podspec is NOT to be published. It is only used as a local source!
```

Comment:
"This podspec is NOT to be published. It is only used as a local source!"
You disagree peter?